app/src/interfaces/many-to-many/options.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+	<v-notice type="warning" v-if="collection == null">
```

Comment:
Another one @Nitwel :P 

```suggestion
	<v-notice type="warning" v-if="collection === null">
```